src/main/java/com/microsoftopentechnologies/windowsazurestorage/helper/AzureUtils.java

```diff
@@ -203,6 +238,40 @@ public static SharedAccessFilePolicy generateFilePolicy() {
         return policy;
     }
 
+    /**
+     * Generates SAS URL for file item in Azure storage File Share.
+     *
+     * @param storageAccount
+     * @param fileName
+     * @param shareName      container name
+     * @return SAS URL
+     * @throws Exception
+     */
+    public static String generateFileSASURL(
+            StorageAccountInfo storageAccount,
+            String shareName,
+            String fileName,
+            EnumSet<SharedAccessFilePermissions> permissions) throws Exception {
+        CloudStorageAccount cloudStorageAccount = getCloudStorageAccount(storageAccount);
+
+        CloudFileClient fileClient = cloudStorageAccount.createCloudFileClient();
+        CloudFileShare fileShare = fileClient.getShareReference((shareName));
+        if (!fileShare.exists()) {
```

Comment:
same as above, suggest to auto create fileshare if not exists.

Reply:
Same as above